colossus-tests/src/test/scala/colossus/Util.scala

```diff
@@ -40,6 +41,33 @@ object RawProtocol {
     type Input = ByteString
     type Output = ByteString
   }
+  implicit object ServiceClientLifter extends ServiceClientLifter[Raw, RawClient[Callback]] {
```

Comment:
Can we call this the CallbackClientLifter so that its name jives with the FutureClientLifter?